src/Commands/SystemCommands/GenericmessageCommand.php

```diff
@@ -23,13 +24,36 @@ class GenericmessageCommand extends SystemCommand
     protected $name = 'Genericmessage';
     protected $description = 'Handle generic message';
     protected $version = '1.0.1';
+    protected $need_mysql = true;
     /**#@-*/
 
     /**
-     * {@inheritdoc}
+     * Execution if MySQL is required but not available
+     *
+     * @return boolean
      */
-    /*public function execute()
+    public function executeNoDB()
     {
+        //Do nothing
+        return true;
```

Comment:
Must return `ServerResponse`.

(See [next inline comment](https://github.com/akalongman/php-telegram-bot/pull/104/files#r54173944) for more info)